### ISSUE TITLE
Machine learning and its applications to biology.

### ISSUE THREAD

**Issue by pschloss** [SchlossLab/march_madness_2015#53]
Machine learning and its applications to biology.
Tarca AL, Carey VJ, Chen XW, Romero R, Drăghici S.
PLoS Comput Biol. 2007 Jun;3(6):e116. Review. No abstract available.
http://www.ncbi.nlm.nih.gov/pubmed/17604446

**Comment by nbaxter13** [comment]
Tarca and friends give an excellent review of machine learning algorithms used for biological datasets.  It’s clearly written by bioinformagicians for biologists, and they do a reasonably good job of dumbing it down to our level.  They start with simple explanations of algorithms like decision trees, neural networks, and SVMs.  They have good illustrative figures so that you can completely gloss over the mathy junk and still understand what they're talking about (except for neural networks, which still make no sense.  WTF is this "hidden layer" stuff?).  This section ends with an explanation of dimensionality reduction, which is extremely important for our high p low n datasets.  Then there’s some mildly interesting stuff about clustering algorithms, followed a by the final section which shows a few examples of these algorithms implemented in R.  This section seems like more of a sales pitch for R than anything else, but it does show some interesting ways of evaluating models that I think we could learn from.

Overall this is a great review of machine learning algorithms that’s seems written for people just like us.  I strongly recommend that everyone in the lab read it, and look back at it from time to time.  For me it was a good reminder of the other techniques that we could use, and it gave me some very practical advice for ways to improve my analyses.

tl;dr – Kathryn finally picked something interesting.  You should totally read it. Just ignore the math.

Overall – 9
Importance – 8
Innovation – 5
